### DISCUSSION TITLE
Using a graphql project [Apr 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1633] [Discord Threads]
> 🙋‍♂️ **Cris Acosta**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### what is the issue you are facing  
#  
> 🙋‍♂️ **Cris Acosta**  
#### Hi!  
#  
> 🙋‍♂️ **Cris Acosta**  
#### I installed on Heroku. I watched the video but didn't find the same menu as the video. No, I don't know how to start.  
#  
> 💁 **o1lab**  
#### you can go to home screen,    
#### and create a new project   
#### it will ask you to choose which api type to choose (2nd option) 🙂  
#  
> 🙋‍♂️ **Cris Acosta**  
#### localhost:8080 refused  
#  
> 👨‍🚀 **wingkwong**  
#### if you are using heroku, then you should access via    
####    
#### https://<YOUR_APP_NAME>.herokuapp.com/dashboard  
#  
> 👨‍🚀 **wingkwong**  
#### or simply click Open app in Heroku GUI  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/959077315382489130)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
